test/gst-msdk/encode/mpeg2.py

```diff
@@ -48,8 +48,8 @@ def test_cqp(case, gop, bframes, qp, quality, profile):
     slash.skip_test("{} profile is not supported".format(profile))
 
   params.update(
-    gop = gop, bframes = bframes, qp = qp, quality = quality,
-    profile = mprofile, mformat = mapformat(params["format"]))
+    gop = gop, bframes = bframes, qp = mapRange(qp, [0, 100], [0, 51]),
```

Comment:
For modified/mapped variable names, please prefix with "`m`".  That is, use "`mqp`" instead of "`qp`" for the mapped qp result.  This makes it clearer that they are different.  We want to use the original/normalized value (`qp`) in, for example, the artifact filename.  But use the mapped value (`mqp`) in the pipeline command.